packages/env/lib/env.js

```diff
@@ -15,6 +15,12 @@ const createDockerComposeConfig = require( './create-docker-compose-config' );
 const detectContext = require( './detect-context' );
 const resolveDependencies = require( './resolve-dependencies' );
 
+/**
+ * Promisified dependencies
+ */
+const copyDir = util.promisify( require( 'copy-dir' ) );
+const wait = util.promisify( setTimeout );
```

Comment:
TIL `setTimeout[ require( 'util' ).promisify.custom ]` is a thing.

Reply:
Yeah, I think it used to be that you had to use `setTimeout[ require( 'util' ).promisify.custom ]` directly.